jekyll/_cci2/testing-ios.md

```diff
@@ -503,15 +503,15 @@ A series of simulator-related issues are known to happen on some
 projects. Here are the most frequent of those:
 
 * **Xcode version is not available.** We install
-  a few different versions of Xcode in the {% comment %} TODO: Job {% endcomment %}build
+  a few different versions of Xcode in each build
```

Comment:
I assume this falls into the same bucket of "you have a good reason to leave this as build," but then I'm confused why all job runs aren't builds

Reply:
I'm not clear why this is staying as Build

Reply:
This is used as part of "build image", which I think is a specific thing that will not turn into "job image".